scripts/tables.R

```diff
@@ -0,0 +1,1439 @@
+#          road_class, road_name_full, road_surface, file_type_description, forest_file_id,
+#          client_name, client_name_abb, map_label, owner_name, admin_area_abbreviation,
+#          steelhead_network_km, steelhead_belowupstrbarriers_network_km, distance) %>%
+#   # filter(distance < 100) %>% ## we need to screen out the crossings that are not matched well
+#   select(pscis_crossing_id, my_crossing_reference:admin_area_abbreviation, steelhead_network_km, steelhead_belowupstrbarriers_network_km)
+
+
+
+####-----------report table--------------------
+#  HACK hashout for now!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! becasue some columns are now missing from bcfishpass.crossings
+
+# tab_cost_rd_mult <- readr::read_csv(paste0(getwd(), '/data/inputs_raw/tab_cost_rd_mult.csv'))
+#
+# tab_cost_rd_mult_report <- tab_cost_rd_mult %>%
+#   mutate(cost_m_1000s_bridge = cost_m_1000s_bridge * 10) %>%
+#   rename(
+#     Class = my_road_class,
+#     Surface = my_road_surface,
+#     `Class Multiplier` = road_class_mult,
+#     `Surface Multiplier` = road_surface_mult,
+#     `Bridge $K/10m` = cost_m_1000s_bridge,
+#     `Streambed Simulation $K` = cost_embed_cv
+#   ) %>%
+#   filter(!is.na(Class)) %>%
+#   mutate(Class = stringr::str_to_title(Class),
+#          Surface = stringr::str_to_title(Surface)
+#   )
+#
+# pscis_rd <- left_join(
+#   rd_class_surface,
+#   xref_pscis_my_crossing_modelled,
+#   by = c('my_crossing_reference' = 'external_crossing_reference')
+# ) %>%
+#   mutate(stream_crossing_id = as.numeric(stream_crossing_id)) %>% #should be able to remove this after we have the data in?
+#   mutate(pscis_crossing_id = case_when(!is.na(stream_crossing_id) ~ stream_crossing_id,
+#                                        T ~ pscis_crossing_id)) %>%
+#   select(-stream_crossing_id)
+#   # filter(distance < 100)
+#  HACK bottom hashout for now!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+
+
+
+
+
+
+
+#------------------make the tables for the methods---------- this should be added to fpr data-raw
+tab_habvalue <- tibble::tibble(`Habitat Value` = c('High', 'Medium', 'Low'),
+                                `Fish Habitat Criteria` = c(
+                                  'The presence of high value spawning or rearing habitat (e.g., locations with abundance of suitably sized gravels, deep pools, undercut banks, or stable debris) which are critical to the fish population.',
+                                  'Important migration corridor. Presence of suitable spawning habitat. Habitat with moderate rearing potential for the fish species present.', 'No suitable spawning habitat, and habitat with low rearing potential (e.g., locations without deep pools, undercut banks, or stable debris, and with little or no suitably sized spawning gravels for the fish species present).'
+                                )
+)
+
+
+
+
+tab_barrier_scoring <- dplyr::tribble(
```

Comment:
dont forget that we don't need these anymore because they are fpr objects. you should replace inline and delete here